libshaderc/src/shaderc_cpp_test.cc

```diff
@@ -30,6 +30,13 @@ const char kMinimalShader[] = "void main(){}";
 const std::string kMinimalShaderWithMacro =
     "#define E main\n"
     "void E(){}\n";
+const std::string kDeprecatedAttributeShader =
+    "#version 130\n"
+    "attribute float x;\n"
```

Comment:
Put the comment from line 339 here, so the reader will always see it, no matter which method uses this.